packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -369,6 +370,114 @@ interface IFileInfo {
     rawData: Nullable<ArrayBufferView>;
 }
 
+/**
+ * Defines options for SceneLoader plugins. This interface is extended by specific plugins.
+ */
+export interface SceneLoaderPluginOptions extends Record<string, Record<string, unknown> | undefined> {}

```

Comment:
Maybe on a separate file?

Reply:
Do you mean just this one interface, or all the interfaces I added that are options related? My preference would be to leave them here as it seems simpler for the user (they are just importing SceneLoader related types from a single file). If you feel strongly that this one (and/or others) should be moved to another file, will you share a little more of your thinking?

Reply:
Chatted offline, we can follow up on this for a later PR if you'd like!